src/main/resources/templates/admin/email.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE HTML>
```

Comment:
Is this page linked from anywhere?

Reply:
It is now linked from the main admin page.